Makefile

```diff
@@ -10,7 +10,7 @@ check-gofmt:
 	scripts/check_gofmt.sh
 
 test:
-	go test ./...
+	go test --race ./...
```

Comment:
Go tends to prefer the BSD-style flags (i.e. one hyphen instead of two, and no short names) for its executables, so let's use `-race` instead of this [as described here](https://blog.golang.org/race-detector). I'm actually fairly surprised that `--race` even works!